doc/manual/src/command-ref/nix-collect-garbage.md

```diff
@@ -1,24 +1,49 @@
 # Name
 
-`nix-collect-garbage` - delete unreachable store paths
+`nix-collect-garbage` - delete unreachable [store objects]
 
 # Synopsis
 
 `nix-collect-garbage` [`--delete-old`] [`-d`] [`--delete-older-than` *period*] [`--max-freed` *bytes*] [`--dry-run`]
 
 # Description
 
-The command `nix-collect-garbage` is mostly an alias of [`nix-store
---gc`](@docroot@/command-ref/nix-store/gc.md), that is, it deletes all
-unreachable paths in the Nix store to clean up your system. However,
-it provides two additional options: `-d` (`--delete-old`), which
-deletes all old generations of all profiles in `/nix/var/nix/profiles`
-by invoking `nix-env --delete-generations old` on all profiles (of
-course, this makes rollbacks to previous configurations impossible);
-and `--delete-older-than` *period*, where period is a value such as
-`30d`, which deletes all generations older than the specified number
-of days in all profiles in `/nix/var/nix/profiles` (except for the
-generations that were active at that point in time).
+The command `nix-collect-garbage` is mostly an alias of [`nix-store --gc`](@docroot@/command-ref/nix-store/gc.md),
+that is, `nix-collect-garbage` deletes all unreachable [store objects] in the Nix store to clean up your system.
+
+However, it provides two additional options which also delete old [profiles], allowing potentially more [store objects] to be deleted because profiles are also garbage collection roots.
+These options are the equivalent of running variations of [`nix-env --delete-generations`](@docroot@/command-ref/nix-env/delete-generations.md)
+on various profiles, prior to running `nix-collect-garbage` (or just `nix-store --gc`) without any flags.
```

Comment:
I mentioned "variations" because the argument to that one is different depending on the flag/argument to this command.

Perhaps there is a better way to say this, like "running <cmd> with various arguments"?

Reply:
```suggestion
These options are the equivalent of running
[`nix-env --delete-generations`](@docroot@/command-ref/nix-env/delete-generations.md)
with various augments on multiple profiles,
prior to running `nix-collect-garbage` (or just `nix-store --gc`) without any flags.
```